app/views/players/show.html.erb

```diff
@@ -33,10 +33,10 @@
     <div class="col-md-4"></div>
     <div class="col-md-4 text-center">
       <% if policy(@player).update? %>
-        <%= link_to "Edit Player", edit_team_player_path(@team, @player), class: 'edit btn btn-success' %>
+        <%= link_to "Edit Player", edit_player_path(@player), class: 'edit btn btn-success' %>
       <% end %>
       <% if policy(@player).destroy? %>
-        <%= link_to "Delete Player", [@team, @player], method: :delete, class: 'cancel btn btn-danger', data: { confirm: 'Are you sure you want to delete this player?' } %>
+        <%= link_to "Delete Player", [@player], method: :delete, class: 'cancel btn btn-danger', data: { confirm: 'Are you sure you want to delete this player?' } %>
```

Comment:
No need to wrap `[@player]` in an `Array` when there's only a single object being passed to `link_to`.